package.json

```diff
@@ -14,7 +14,8 @@
     "react": "18.0.0",
     "react-bootstrap": "^2.2.2",
     "react-dom": "18.0.0",
-    "react-icons": "^4.3.1"
+    "react-icons": "^4.3.1",
+    "sass": "^1.54.9"
```

Comment:
> I did npm install sass. So, there may be some unwanted dependencies installed. I'm not too sure which dependencies are from sass.

Only sass was added, so you're good! Don't worry about the extra stuff in the .lock files